docs/components/dartpad/switches/regular/main.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      debugShowCheckedModeBanner: false,
+      home: MyHomePage(),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  @override
+  _MyHomePageState createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  List<bool> _values = [true, false, true, false, false];
+
+  Widget build(BuildContext context) {
+    final int count = 4;
+
+    return Scaffold(
+      body: Column(
+        children: <Widget>[
+          for (int i = 0; i <= count; i++)
+            ListTile(
+              title: Text(
+                'Switch ${i+1}',
```

Comment:
Did you run `dartfmt` on this code? I would have thought that it would format this as `i + 1`

Reply:
good catch, formatted both demos and readme